.opencode/skills/dbt-troubleshoot/SKILL.md

```
@@ -155,6 +182,119 @@ altimate-dbt execute --query "SELECT * FROM {{ ref('<name>') }}" --limit 10
+    description: <one-line description from the prompt or your understanding>
+    columns:
+      - name: <pk_or_first_column>
+```
+
+A minimal `name:` entry satisfies structural "model is registered" checks even without descriptions or column-level tests.
+
+### Step 5d: Pre-finish hard-stop checklist (mandatory)
+
+Before declaring done, **echo the following checklist explicitly** (one line each, with the answer):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)